runtime/help/plugins.md

```diff
@@ -31,48 +33,62 @@ of which there are 5 types:
 * Syntax header files
 
 In most cases, a plugin will want to add help files, but in certain
-cases a plugin may also want to add colorschemes or syntax files. It
-is unlikely for a plugin to need to add plugin files at runtime or
-syntax header files. No directory structure is enforced but keeping
-runtime files in their own directories is good practice.
+cases, a plugin may also want to add colorschemes or syntax files. It
+is unlikely for a plugin to need to add plugin or syntax header files at
+runtime. No directory structure is enforced, but keeping runtime files in their
+own directories is good practice.
 
 ## Lua callbacks
 
-Plugins use Lua but also have access to many functions both from micro
-and from the Go standard library. Many callbacks are also defined which
-are called when certain events happen. Here is the list of callbacks
-which micro defines:
+Plugins use Lua but also have access to many functions, both from micro
+and from the Go standard library. Plugins can also define functions that micro
+will call when certain events happen.
+Here is the list of callbacks that micro defines:
+
+* `init()`:
+    this function should be used for your plugin initialization. This function
+    is called after buffers have been initialized.
+
+* `preinit()`:
+    initialization function called before buffers have been initialized.
 
-* `init()`: this function should be used for your plugin initialization.
-   This function is called after buffers have been initialized.
+* `postinit()`:
+    initialization function called after `init()`.
 
-* `preinit()`: initialization function called before buffers have been
-   initialized.
+* `deinit()`:
+   cleanup function called when your plugin is unloaded, either manually by the
+   user, or by running the `reload` command.
 
-* `postinit()`: initialization function called after `init()`.
+* `onSetActive(`[bufpane][BufPane]`)`:
```

Comment:
Again, a user reading `help plugins` in micro (in plain text) will have no idea what the heck is this `[bufpane][BufPane]`.

Reply:
Having the links in the rendered markdown is extremely useful though so maybe the tradeoff is worth it?

Reply:
I think it is not a tradeoff but the opposite of it, i.e. favoring one use case at the expense of the other one.

A tradeoff would be something like: do add these links, but only as "footnotes". Perhaps also with a sentence like `See these links for documentation on these data structures:`.